engine/app.py

```diff
@@ -18,10 +18,10 @@ async def main(websocket):
         endMove = moves[2]
 
         # Do move and update board
-        if(board.can_move_piece(startMove, endMove)):
-            board.move_piece(startMove, endMove)
-            
-            # Send resulting board/game state
-            await websocket.send(board.game_to_json())            
+        #if(board.can_move_piece(startMove, endMove)):
+        board.move_piece(startMove, endMove)
+        
+        # Send resulting board/game state
+        await websocket.send(board.game_to_json())            
```

Comment:
@GarettOS I had to comment out the move validation to get this PR to work, but its pretty cool to be able to move all of the pieces around right now